drive/src/common/mod.rs

```diff
@@ -14,15 +14,20 @@ use crate::drive::Drive;
 use crate::error::structure::StructureError;
 use crate::error::Error;
 
+use dpp::data_contract::extra::DriveContractExt;
```

Comment:
Shouldn't be drive contract ext placed in drive crate?

Reply:
Yeah, I've been thinking about it also. I placed it in the DPP because the `DriveContractExt` API should eventually be blended with the standard API. So the functional ownership belongs to the DPP  - not to the Drive. DPP knows about Drive only throughout the name of trait, so it's not an actual dependency. Additionally, the `DriveContractExt` implementation uses private fields which should stay private to the DPP consumers.

Reply:
It's just the extensions for now in dpp that are needed for drive, eventually they will be merged into dpp.